src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseApplicantNotificationHandler.java

```diff
@@ -116,9 +116,13 @@ private void sendNotificationToSolicitor(CaseData caseData, String recipient) {
 
     private void sendNotificationToSolicitorSpec(CaseData caseData, String recipient, CaseEvent caseEvent) {
         String emailTemplate;
-        emailTemplate = caseEvent.equals(NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CC)
-            ? notificationsProperties.getRespondentSolicitorDefendantResponseForSpec()
-            : notificationsProperties.getClaimantSolicitorDefendantResponseForSpec();
+        if (caseEvent.equals(NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CC)) {
+            emailTemplate = notificationsProperties.getRespondentSolicitorDefendantResponseForSpec();
+        } else if (caseEvent.equals(NOTIFY_RESPONDENT_SOLICITOR2_FOR_DEFENDANT_RESPONSE_CC)) {
+            emailTemplate = notificationsProperties.getRespondentSolicitorDefendantResponseForSpec();
```

Comment:
since the value is the same, use NOTIFY_APPLICANT_SOLICITOR1_FOR_DEFENDANT_RESPONSE_CC.equals(caseEvent) || NOTIFY_RESPONDENT_SOLICITOR2_FOR_DEFENDANT_RESPONSE_CC.equals(caseEvent)